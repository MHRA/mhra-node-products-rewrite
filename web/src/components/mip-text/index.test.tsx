import renderer from 'react-test-renderer';
import React from 'react';
import MipText from './index';

describe(MipText, () => {
  it('should render', () => {
    const component = renderer.create(<MipText />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
