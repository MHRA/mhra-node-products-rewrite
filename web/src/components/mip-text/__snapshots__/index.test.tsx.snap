// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MipText should render 1`] = `
<section
  className="sc-bczRLJ fqyYGh"
  dangerouslySetInnerHTML={
    Object {
      "__html": Object {
        "process": [Function],
      },
    }
  }
/>
`;
